web-basics-browser-security-model/activities/cockroach/README.md

```diff
@@ -0,0 +1,17 @@
+# Name
+
+Web: Web basics and browser security model: Cockroach
+
+## Description
+
+Get the flag from [cockroach](http://141.85.224.157:8080/cockroach/). What happened? Get the flag!
```

Comment:
URL does not work

Reply:
I won't fix this problem now, after we have a decision on Discord we can move forward with that solution.